### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by sameeul** [conda-forge/schema-salad-feedstock#126]
### Comment:

Hi, I am curious why the recipe skips Windows build. I see that there are PyPI wheels for windows, so I am curious to know what technical difficulty is preventing us from building a Windows package for conda.

I have a downstream project that depends on this package (via transitive dependency). I am also open to help to get the Windows package build.

**Comment by mr-c** [comment]
Hello @sameeul ; can you share more about this dependency tree that include `schema-salad`?

There are no binary wheels for `schema-salad` on PyPI, just a source wheel: https://pypi.org/project/schema-salad/8.7.20241021092521/#files

**Comment by sameeul** [comment]
We are working on creating a recipe for this package (https://github.com/PolusAI/workflow-inference-compiler/). It uses `cwl-utils` and that brings `schema-salad` as a dependency.

**Comment by mr-c** [comment]
@sameeul Note that `cwltool` does not support running in MS Windows, but it is supported inside WSL2: https://cwltool.readthedocs.io/en/latest/#ms-windows-users

**Comment by sameeul** [comment]
Yes, we are aware of it. We are also developing another "compute" tool that can take CWL workflows generated by this tool and can run in various compute environment (e.g.. on HPC using toil, on Nix systems using cwltool, and in AWS EC2 using argo)

**Comment by mr-c** [comment]
@sameeul The schema-salad tests don't currently pass on MS Windows; that is probably fixable. Can you contribute a PR? 

https://github.com/common-workflow-language/schema_salad/actions/runs/11499085946/job/32006276340?pr=887

**Comment by sameeul** [comment]
Sure. I will take a stab at it.

**Comment by mr-c** [comment]
> Sure. I will take a stab at it.

Ping from 2025 :-)
